monkey/tests/unit_tests/infection_monkey/network_scanning/test_tcp_scanner.py

```diff
@@ -38,32 +66,48 @@ def test_tcp_successful(
         assert port_scan_data[port].status == PortStatus.CLOSED
         assert port_scan_data[port].banner is None
 
+    event = _get_tcp_scan_event(port_scan_data)
```

Comment:
I'm also wondering if it would be better to use `call_args` even though the syntax is gross. Something about having  code to build a TCPScanEvent from PortScanData feels wrong